src/transformers/models/llava/processing_llava.py

```diff
@@ -52,21 +52,21 @@ class LlavaProcessor(ProcessorMixin):
     """
 
     attributes = ["image_processor", "tokenizer"]
-    valid_kwargs = ["chat_template", "patch_size", "vision_feature_select_strategy", "image_token"]
+    valid_kwargs = ["chat_template", "num_image_tokens", "vision_feature_select_strategy", "image_token"]
     image_processor_class = "AutoImageProcessor"
     tokenizer_class = "AutoTokenizer"
 
     def __init__(
         self,
         image_processor=None,
         tokenizer=None,
-        patch_size=None,
```

Comment:
Won't this thrown an error if this argument is removed, was previously part of a saved processors config and now isn't a valid kwarg?

Reply:
no, the prev checkpoints were not touched yet and afaik the new logic isn't used anywhere